cmd/contour/serve.go

```diff
@@ -169,7 +166,6 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 }
 
 type Server struct {
-	group      workgroup.Group
```

Comment:
looks like we could almost drop the `workgroup` package entirely, but it's still used in the featuretests framework. We could do the work to fully drop it in a separate PR.

Reply:
yeah, didnt drop it entirely in case we needed it for some reason, could just use some simple coordination mechanisms to run components in tests w/o the need for the workgroup code

Reply:
https://github.com/projectcontour/contour/issues/4206